app/api/client.py

```diff
@@ -109,6 +109,6 @@ def execute_mutation(
         result = self.execute_gql(document, variable_values=variable_values)
         data = result[name]
         if not data["ok"]:
-            raise GraphQLMutationError(f"{name} mutation failed.", data.get("error"))
+            raise GraphQLMutationError(name, data.get("error"))
```

Comment:
Bugfix - `GraphQLMutationError` expects to get the operation name as first parameter not a message